Makefile

```diff
@@ -185,107 +184,244 @@ test-integration: test-integration-aws test-integration-clickhouse test-integrat
 test-integration: test-integration-gcp test-integration-influxdb test-integration-kafka test-integration-loki
 test-integration: test-integration-pulsar test-integration-splunk
 
+stop-test-integration: ## Stops all integration test infrastructure
+stop-test-integration: stop-integration-aws stop-integration-clickhouse stop-integration-elasticsearch
+stop-test-integration: stop-integration-gcp stop-integration-influxdb stop-integration-kafka stop-integration-loki
+stop-test-integration: stop-integration-pulsar stop-integration-splunk
+
+start-integration-aws:
+	$(CONTAINER_TOOL) network create test-integration-aws
+	$(CONTAINER_TOOL) run -d --network=test-integration-aws -p 8111:8111 --name ec2_metadata \
+	 timberiodev/mock-ec2-metadata:latest
+	$(CONTAINER_TOOL) run -d --network=test-integration-aws -p 4568:4568 -p 4572:4572 -p 4582:4582 -p 4571:4571 -p 4573:4573 \
+	 --name localstack -e SERVICES=kinesis:4568,s3:4572,cloudwatch:4582,elasticsearch:4571,firehose:4573 \
+	 localstack/localstack@sha256:f21f1fc770ee4bfd5012afdc902154c56b7fb18c14cf672de151b65569c8251e
+	$(CONTAINER_TOOL) run -d --network=test-integration-aws -p 6000:6000 --name mockwatchlogs \
+	 -e RUST_LOG=trace luciofranco/mockwatchlogs:latest
+	sleep 5
+
+stop-integration-aws:
+	$(CONTAINER_TOOL) rm --force ec2_metadata mockwatchlogs localstack 2>/dev/null; true
+	$(CONTAINER_TOOL) network rm test-integration-aws 2>/dev/null; true
+
 test-integration-aws: ## Runs AWS integration tests
```

Comment:
This fails locally because I don't have `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` set.

Do you think we could prevent this with something like:

```makefile
test-integration-aws: AWS_ACCESS_KEY_ID ?= "dummy"
test-integration-aws: AWS_SECRET_ACCESS_KEY ?= "dummy"
test-integration-aws: ## Runs AWS integration tests
# ...
```

Reply:
Good catch - yeah it works for me because I have those.